test/test_data/data_5bus_pu.jl

```diff
@@ -598,20 +601,39 @@ reserve5(thermal_generators5) = [
         0.8,
         maximum([gen.activepowerlimits[:max] for gen in thermal_generators5]) .* 0.001,
     ),
+    ReserveDemandCurve{ReserveUp}("ORDC1", true, 0.6, ORDC_cost),
 ]
 
 reserve5_re(renewable_generators5) = [
     VariableReserve{ReserveUp}("Reserve3", true, 30, 100),
     VariableReserve{ReserveDown}("Reserve4", true, 5, 50),
+    ReserveDemandCurve{ReserveUp}("ORDC2", true, 0.6, ORDC_cost),
 ]
 reserve5_hy(hydro_generators5) = [
     VariableReserve{ReserveUp}("Reserve5", true, 30, 100),
     VariableReserve{ReserveDown}("Reserve6", true, 5, 50),
+    ReserveDemandCurve{ReserveUp}("ORDC3", true, 0.6, ORDC_cost),
 ]
 
 reserve5_il(interruptible_loads) = [
     VariableReserve{ReserveUp}("Reserve7", true, 30, 100),
     VariableReserve{ReserveDown}("Reserve8", true, 5, 50),
+    ReserveDemandCurve{ReserveUp}("ORDC3", true, 0.6, ORDC_cost),
+]
+
+function make_ordc_cost(cost::TwoPartCost)
+    var_cost = PSY.get_cost(PSY.get_variable(cost))
+    flatten_array = Array(collect(Iterators.flatten(var_cost))')
```

Comment:
I don't follow this.  `name` is an array but `flatten_array` is an array of arrays.  The reason isn't obvious to me.  Is it valid Julia code?  I see a single quote at the end that looks like a bug.

Reply:
What I'm wanted to do was convert a array of tuples 
```julia
 var_cost = PSY.get_cost(PSY.get_variable(ORDC_cost))
5-element Array{Tuple{Float64,Float64},1}:
 (9000.0, 0.0)
 (6000.0, 0.2)
 (500.0, 0.4)
 (10.0, 0.6)
 (0.0, 0.8)
```
to something like this 

```julia
Array(collect(Iterators.flatten(var_cost))')
1×10 Array{Float64,2}:
 9000.0  0.0  6000.0  0.2  500.0  0.4  10.0  0.6  0.0  0.8
```

Reply:
The single quote at the end is just to do a transpose

Reply:
What you have is fine.  I wasn't familiar with that Julia syntax.  Sorrry.